pkg/option/config.go

```diff
@@ -798,6 +798,9 @@ var HelpFlagSections = []FlagsSection{
 			PreAllocateMapsName,
 			BPFCompileDebugName,
 			FragmentsMapEntriesName,
+			EnableBPFClockProbe,
+			EnableBPFMasquerade,
+			EnableIPMasqAgent,
```

Comment:
Although IPMasqAgent is implemented in BPF, I don't really consider it a BPF tweak. @brb any opinion on this?

Reply:
Ok, I can move it back in that case.

Reply:
For now I will move it to others.